tn.sh

```diff
@@ -10,9 +10,8 @@ if type realpath >/dev/null 2>&1 ; then
 fi
 
 # Generate the test keys and certs necessary for the integration tests.
-docker compose run bsetup
+docker compose run --rm bsetup
 
 # Use a predictable name for the container so we can grab the logs later
```

Comment:
This comment ("Use a predictable name...") is outdated as of https://github.com/letsencrypt/boulder/pull/7442/files#diff-21ca8b974325dd146d3cce3b61930165dbb607a86ff1a650a61ccc769cf69439 and can be removed. We deleted the integration tests that were pulling logs from `boulder_tests` because we deleted the `caa-log-checker`.